bin/expressive-migrate-original-messages

```diff
@@ -22,6 +24,6 @@ if (file_exists($a = __DIR__ . '/../../../autoload.php')) {
     exit(1);
 }
 
-$command = new Command($argv[0], new ConsoleHelper());
-$return = $command->process(array_slice($argv, 1));
-exit($return);
+$application = new Application('expressive-migrate-original-messages', VERSION);
```

Comment:
This is a BC measure; the goal is to eventually remove it.

Also note that the command names differ between the main `expressive` script and this one.